src/app/sandbox/deploy/LeftSection.tsx

```diff
@@ -96,7 +97,11 @@ export function LeftSection() {
         >
           {isConnected ? 'Deploy' : 'Connect Stacks Wallet'}
         </Button>
-      </HStack>
+        <Text fontSize="xs">
```

Comment:
Let's try to make text semantic, this should be a `p` instead of a `span`

also, please move this text to under the save contract to Hiro platform button